chips/sifive/src/pwm.rs

```diff
@@ -61,6 +61,6 @@ impl Pwm {
         let regs = self.registers;
 
         // Turn the interrupt compare off so we don't get any RTC interrupts.
-        regs.cfg.write(cfg::enalways::CLEAR + cfg::enalways::CLEAR);
+        regs.cfg.write(cfg::enalways::CLEAR + cfg::enoneshot::CLEAR);
```

Comment:
Should this possibly be both lines rather than a replacement (clear the 'always' (level?) interrupt and the 'oneshot' interrupt)?

Reply:
Before it was setting the same bit twice, and I think my intention was to clear both bits.

Reply:
Wow..., yes, yes it was, I can read.

Would be interesting to see whether such an error could be automatically caught, it's meaningless to 'add' a bit to itself in a mask like it was.